yaptide/batch/watcher.py

```diff
@@ -30,9 +36,37 @@ def log_generator(thefile, timeout: int = 3600) -> str:
         yield line
 
 
-def read_file(filepath: Path, job_id: str, task_id: int):  # skipcq: PYL-W0613
+def send_task_update(sim_id: int, task_id: str, update_key: str, update_dict: dict) -> bool:
```

Comment:
I'd add an URL of backend as a parameter to this method. This would simplify testing.